src/OpenLoco/GameCommands/Cheat.cpp

```diff
@@ -162,18 +162,13 @@ namespace OpenLoco::GameCommands
                     continue;
 
                 Vehicles::Vehicle train(vehicle);
-                // train.head->reliability = newReliablity;
-                // train.veh1->reliability = newReliablity;
                 train.veh2->reliability = newReliablity;
-                // train.tail->reliability = newReliablity;
 
                 for (auto& car : train.cars)
                 {
                     for (auto& component : car)
                     {
                         component.front->reliability = newReliablity * 256;
```

Comment:
this is still not quite correct.
We want to only be setting the reliability of the first car components front bogie. This sets the front bogie of all car components.
![trainstruct](https://user-images.githubusercontent.com/1277401/115136500-0f393a00-a018-11eb-833b-0a64c9920bee.png)

So you need to break the `for(auto& component : car)` after setting the front.

```
*(car.begin()).front->reliability = newReliability * 256
```
Might need to play about with that a little i've never had to do it so far.

Reply:
I didn't realise just setting the first car's front bogie would be enough. Hope my latest changes are what you had in mind? They seem to be work correctly, at least.